README.md

```diff
@@ -56,58 +56,73 @@ We currently do not:
+docker-compose -f ci/docker-compose-local.yml build app && docker-compose -f ci/docker-compose-local.yml up
 ```
 
-This also applies all migrations to the database container, ci_db_1.  To see useful flags that you might want to use with the `up` subcommand, run `docker-compose up --help`.  This docker-compose command creates the container ci_app_1, among others.
-
 If AWS SES is enabled as a provider, you may need to run the following command to give the (simulated) SES permission to (pretend to) send e-mails:
 
 ```bash
 aws ses verify-email-identity --email-address stage-notifications@notifications.va.gov --endpoint-url=http://localhost:4566
 ```
 
-To support running locally, the repository includes a default `app/version.py` file, which must be present at runtime to avoid raising ImportError.  The production container build process overwrites this file with current values.
+To support running locally, the repository includes a default `app/version.py` file, which must be present at runtime to avoid raising an `ImportError`.  The deployed container build process overwrites this file with current values.
 
 ### Creating database migrations
 
 Running `flask db migrate` on the container ci_app_1 errors because the files in the migrations folder are read-only.  Follow this procedure to create a database migration using Flask:
 
-1. Ensure all containers are stopped.
-2. Run `docker-compose -f ci/docker-compose-local-migrate.yml up`.  This creates the container ci_app_migrate with your local notification-api directory mounted in read-write mode.  The container runs `flask db migrate` and exits.
-3. Press Ctrl-C to stop the containers, and identify the new file in migrations/versions.  (Running `git status` is a quick way to do this.)  Rename and edit the new file as desired.
+1. Ensure all containers are stopped and that the notification_api image has been built
+2. Run `docker compose -f ci/docker-compose-local-migrate.yml up`.  This creates the container ci_app_migrate with your local notification-api directory mounted in read-write mode.  The container runs `flask db migrate` and exits.
+3. Press Ctrl-C to stop the containers, and identify the new file in `migrations/versions/`.
 
 ### Unit testing
 
-Build the "ci_test" Docker image by running this command:
+Build and test the "ci_test" Docker image by running this command:
+
+```bash
+docker compose -f ci/docker-compose-test.yml up
+```
+
+**Rebuild ci_test whenever Dockerfile or poetry.lock changes.**
 
+For a more interactive testing experience, edit `scripts/run_tests.sh` so that it does not execute any pytest command, and place `tail -f` on the final line e.g.
 ```bash
-docker-compose -f ci/docker-compose-test.yml build test
+params="-rfe --disable-pytest-warnings --cov=app --cov-report=term-missing --junitxml=test_results.xml -q"
+# pytest ${params} -n auto -m "not serial" tests/ && pytest ${params} -m "serial" tests/
+display_result $? 2 "Unit tests"
+tail -f
```

Comment:
This duplicates information.  I would instead direct the reader to tests/README.md.

Reply:
Will address in #1692